conanfile.py

```diff
@@ -22,13 +22,13 @@ class RestinioConan(ConanFile):
     build_policy = "missing"
 
     def requirements(self):
-        self.requires.add("http-parser/2.8.1@bincrafters/stable")
+        self.requires.add("http_parser/2.9.2")
```

Comment:
Is this a supported version change?

Reply:
We're using http-parser v.2.9.2 during the development of RESTinio for several month: https://github.com/Stiffstream/restinio/blob/master/externals.rb#L21